### PR TITLE
Milestone 04: Controllers

### PR DESCRIPTION
**Pull request by mavericks-db**
## Controllers

**In this PR, I implement the following:**
- [x] Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes:

- [x] For each URL you should create:
  - A route.
  - An action in the correct controller.
  - A view file.
- [x] View files should include only placeholders written in plain HTML, e.g:
  - Here is a list of posts for a given user
- [x] Make sure that you added code to handle only the 4 required URLs.

routes

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+  get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get 'users/:id', to: 'users#show', as: 'user'
+  get 'users', to: 'users#index', as: 'users'
```
Comment (zhadier) [inline config/routes.rb:5]: - [ ] [ _Optional_ ] Another way to implement restful routes is by using the resources keyword provided by Rails. You can choose to explore this as it is much more easier to implement
``` ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

**Review by zhadier — APPROVED** [review]
## Hi @mavericks-db  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (mavericks-db) [reply config/routes.rb:5]: Thanks, @zhadier for this suggestion.

**Pull request merged**